administrator/components/com_config/tmpl/application/default.php

```diff
@@ -31,12 +31,12 @@
 	<div class="row">
 		<!-- Begin Sidebar -->
 		<div id="sidebar" class="col-md-2">
-            <button class="navbar-toggler options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
-                 <span class="burger-toggler-icon">
+            <button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
```suggestion
            <button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>">
```

Use language strings not hardcoded english please